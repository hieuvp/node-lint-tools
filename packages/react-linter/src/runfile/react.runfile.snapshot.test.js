const react = require('./react.runfile');

describe('react.runfile', () => {
  it('should match snapshot', () => {
    expect(react).toMatchSnapshot();
  });
});
