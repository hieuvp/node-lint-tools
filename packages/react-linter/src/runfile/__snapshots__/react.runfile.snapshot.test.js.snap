// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react.runfile should match snapshot 1`] = `[Function]`;
